File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Lab/Solutions/P09_FruitOrVegetable.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

	string input;
	cin >> input;

	string output;
	if (input == "banana" || input == "apple" || input == "kiwi" || input == "cherry" || input == "lemon" || input == "grapes") {
		output = "fruit";
	} else if (input == "tomato" || input == "cucumber" || input == "pepper" || input == "carrot") {
		output = "vegetable";
	} else {
		output = "unknown";
	}

	cout << output << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Lab/Solutions/P09_FruitOrVegetable.py
name = input()
type = 0

if name == 'banana' \
        or name == 'apple' \
        or name == 'kiwi' \
        or name == 'cherry' \
        or name == 'lemon' \
        or name == 'grapes':
    type = 'fruit'
elif name == 'tomato' \
        or name == 'cucumber' \
        or name == 'pepper' \
        or name == 'carrot':
    type = 'vegetable'
else:
    type = 'unknown'

print(type)